config/crds/migration_v1alpha1_migcluster.yaml

```diff
@@ -58,6 +58,12 @@ spec:
           - isHostCluster
           type: object
         status:
+          properties:
+            observedGeneration:
```

Comment:
This and other CRD changes are just a leftover from artifacts not being checked in on https://github.com/fusor/mig-controller/pull/412/files

Reply:
Don't mind importing them with the PR, as it still requires a CRD change.

Reply:
Ah, looks like I forget the `make manfests` on pull/412.  Let's fix this in a separate PR and then rebase.  The reason for not combining is that if this PR got reverted, we would not want the CRD change to be reverted.

Reply:
Fixed on master: https://github.com/fusor/mig-controller/pull/421